lib/public_suffix.ex

```diff
@@ -59,13 +59,63 @@ defmodule PublicSuffix do
   @spec registrable_domain(String.t, options) :: nil | String.t
   def registrable_domain(domain, options \\ []) when is_binary(domain) do
     # "The registered or registrable domain is the public suffix plus one additional label."
-    parse_domain(domain, options, 1)
+    parse_domain(domain, options, 1, :registrable_or_suffix)
+  end
+
+  @doc """
+  Parses the provided domain and returns the prevailing rule based on the
+  publicsuffix.org rules. If no rules match, the prevailing rule is "*".
+
+  ## Examples
+
+      iex> prevailing_rule("foo.bar.com")
+      "com"
+      iex> prevailing_rule("co.uk")
+      "co.uk"
+      iex> prevailing_rule("foobar.example")
+      "*"
+
+  You can use the `ignore_private` keyword to exclude private (non-ICANN) domains.
+
+      iex> prevailing_rule("foo.github.io", ignore_private: false)
+      "github.io"
+      iex> prevailing_rule("foo.github.io", ignore_private: true)
+      "io"
+      iex> prevailing_rule("foo.github.io")
+      "github.io"
+  """
+  @spec prevailing_rule(String.t) :: nil | String.t
+  @spec prevailing_rule(String.t, options) :: nil | String.t
```

Comment:
Right you are! Which `PublicSuffix.module_info` confirms. I did know about no variable arity, but was confused as to how defaults were handled. Sorry, I didn't mean to nitpick -- I'm learning the language as I go along. Which probably means I should talk less and code more...